array-methods/map.js

```diff
@@ -0,0 +1,7 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+
+const arr1 = prices.map((num) => ({
+  price: num,
+  salePrice: num / 2,
+}));
+console.log(arr1);
```

Comment:
Don't forget the last `.map` task:

---

![image](https://github.com/annaaksenov/m2-0723-code-solutions/assets/16143771/dc65a200-a28a-4c0c-b8e4-65848f579722)